### ISSUE TITLE
Training Duration

### ISSUE THREAD

**Issue by wuxingzhe** [Wi-sc/NVF#2]
Hi, congrats on the great work.  I'm training this work in a 3090 GPU. However, as the number of training epochs is 1500, I need to train for nearly a month. Hence, I feel confused about the training duration。

**Comment by Wi-sc** [comment]
Hi, sorry for the confusion. The training only takes 150~250 epochs, not 1500. You can check if it converges or not according to the validation loss. As long as the validation loss doesn't decrease anymore, you can stop it.
